proto/zitadel/action/v2beta/action_service.proto

```diff
@@ -394,51 +392,123 @@ service ZITADELActions {
 }
 
 message CreateTargetRequest {
-  optional zitadel.object.v3alpha.Instance instance = 1 [
+  string name = 1 [
+    (validate.rules).string = {min_len: 1, max_len: 1000},
     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-      default: "\"domain from HOST or :authority header\""
+      example: "\"ip_allow_list\"";
+      min_length: 1
+      max_length: 1000
     }
   ];
-  Target target = 2 [
-    (validate.rules).message = {
-      required: true
+  // Defines the target type and how the response of the target is treated.
+  oneof target_type {
+    option (validate.required) = true;
+    RESTWebhook rest_webhook = 2;
+    RESTCall rest_call = 3;
+    RESTAsync rest_async = 4;
+  }
```

Comment:
can you make sure there's a curl example that can just be used as is (document one of the oneof examples)

Reply:
What do you want to achieve with this?

Reply:
to the users it's not visible that those fields are actually oneofs and not all can be passed
as defined in the issue, we want to provide at least one easy copy / paste example for users to work properly:
![image](https://github.com/user-attachments/assets/af55f2f2-4532-4d1a-8b85-3ee980e24665)